docs/reference/release_notes.md

```diff
@@ -0,0 +1,145 @@
+---
+myst:
+  html_meta:
+    "description lang=en":
+      "Release notes for Ubuntu on WSL and Ubuntu Pro for WSL."
+---
+
+(ref::release-notes)=
+# Release notes for Ubuntu on WSL
+
+This page includes:
+
+* [Release notes for LTS versions of Ubuntu on WSL](ref::ubuntu-wsl-lts-releases)
+* [Release notes for interim releases of Ubuntu on WSL](ref::ubuntu-wsl-interim-releases)
+* [Release notes for the Ubuntu Pro for WSL application](ref::up4w-releases)
+
+(ref::upgrade-instructions)=
+## Upgrade instructions for Ubuntu on WSL
+
+The default Ubuntu distro for WSL (`wsl --install Ubuntu`) always ships the
+latest stable LTS release, which can be upgraded once the first point release
+of a new LTS is available:
+
+```{code-block} text
+$ sudo apt update && sudo apt full-upgrade 
+$ do-release-upgrade
+```
+
+By default, Ubuntu distros downloaded as explicitly numbered releases (`wsl
+--install Ubuntu-24.04`) will not be upgraded in this manner, unless the
+following configuration change is made:
+
+```{code-block} diff
+:caption: /etc/update-manager/release-upgrades
+:class: no-copy
+- Prompt=lts
++ Prompt=normal
```

Comment:
```suggestion
- Prompt=never
+ Prompt=normal
```

The numbered distros come with the `never` value. Users could change to `lts` or `normal`, which allows upgrading to the next LTS or the next interim release respectively.